mutation/mutation_partition.cc

```diff
@@ -2220,7 +2220,7 @@ stop_iteration reconcilable_result_builder::consume(clustering_row&& cr, row_tom
     }
     _live_rows += is_alive;
```

Comment:
> This pach doesn't address the problem of client-side timeouts during
paging. It is an orthogonal problem. Query pager could periodically
send empty pages to the client, or use some other keep-alive
mechanism.

This implies the coordinator loops around the reconciled pages. But why? Just send an empty result to the caller.

As long as the continue-where cookie makes forward progress, the client will ask for another page, and we'll make forward progress.

Reply:
This suggestion is out-of-scope for the series, it's a big change. But I want @tgrabiec @denesb @gleb-cloudius opinions on it.

Reply:
That would introduce a regression to scans done on behalf of aggregate queries, since now they would involve a round-trip to the client. We could reduce the impact by sending them not at internal boundary, but time boundary related to the timeout. 

I don't know how that fits with the new forward_service.

Reply:
Aggregations (including those by forward_service) are always paged internally.

Reply:
I just realized I forgot to update this commitlog too.

The current version also addresses client-side timeouts, with the exception of aggregates, which are paged internally.
The coordinator itself will not "loop", looking for empty pages anymore, it will send the empty page to the client, just like we do when hitting the tombstone limit during a regular page.